astropy/coordinates/builtin_frames/gcrs.py

```diff
@@ -54,9 +54,19 @@ class GCRS(BaseRADecFrame):
     The frame attributes are listed under **Other Parameters**.
     """
 
-    obstime = TimeAttribute(default=DEFAULT_OBSTIME)
-    obsgeoloc = CartesianRepresentationAttribute(default=[0, 0, 0], unit=u.m)
-    obsgeovel = CartesianRepresentationAttribute(default=[0, 0, 0], unit=u.m / u.s)
+    obstime = TimeAttribute(
```

Comment:
Seeing the repetitions, would it make sense to define `OBSTIME_ATTRIBUTE = TimeAttribute(...)` and repeatedly use it? (Probably best for another PR!!). Equinox might be another candidate.

Reply:
I haven't actually tried this, but I had simply assumed that "reusing" a descriptor like that wouldn't work.

Reply:
I think it should be OK, since the descriptor stores nothing on itself, so it can deal with multiple instances (indeed, it *has* to deal with multiple instances of a given frame class, so as long as the defaults are the same can also deal with multiple instances of multiple frame classes). But it is somewhat out of scope here.

Reply:
Agreed. In `cosmology` we are unnecessarily careful about this for `Ode0` in `FLatFLRW`
https://github.com/astropy/astropy/blob/62aaa295c33a1b653e0a35288f241db8bd89f3ba/astropy/cosmology/flrw/base.py#L1475-L1487

Reply:
Let's also do this in follow-up!